docs/csharp/versioning.md

```diff
@@ -0,0 +1,132 @@
+---
+title: C# Versioning | C# Guide
+description: Understand how versioning works in C# and .NET
+keywords: .NET, .NET Core, C#
+author: tsolarin
+manager: wpickett
+ms.date: 2016/12/03
```

Comment:
This needs to be mm/dd/yyyy format or the published site shows the wrong date. (I know I gave you this format, but it's a bug we've since discovered.)